llvm/lib/MC/MCTargetOptions.cpp

```diff
@@ -16,7 +16,7 @@ MCTargetOptions::MCTargetOptions()
       MCNoWarn(false), MCNoDeprecatedWarn(false), MCNoTypeCheck(false),
       MCSaveTempLabels(false), MCIncrementalLinkerCompatible(false),
       FDPIC(false), ShowMCEncoding(false), ShowMCInst(false), AsmVerbose(false),
-      PreserveAsmComments(true), Dwarf64(false),
+      SSE2AVX(false), PreserveAsmComments(true), Dwarf64(false),
```

Comment:
This will cause initialization reorder warnings on some versions of gcc - make sure it matches the position in the struct

Reply:
Maybe we can drop the change in this file after applying the suggestion

https://github.com/llvm/llvm-project/pull/96860#discussion_r1661919071

`bool SSE2AVX : 1;` to `bool SSE2AVX = false`